app/buckets/routes.js

```diff
@@ -7,5 +7,6 @@ module.exports = [
   { name: 'create', pattern: '/buckets/new', handler: handlers.create_bucket, method: 'POST' },
   { name: 'details', pattern: '/buckets/:id', handler: handlers.bucket_details },
   { name: 'delete', method: 'POST', pattern: '/buckets/:id/delete', handler: handlers.delete },
+  { name: 'access_logs', pattern: '/buckets/:id/access_logs/', handler: handlers.access_logs },
```

Comment:
Not sure if the trailing slash on the pattern is a problem, but none of the other patterns have one, so I'd say remove it for consistency.

Reply:
👍